lab1.js

```diff
@@ -77,12 +82,27 @@ var sentence2 = 'Come over here so you can scratch my belly.';
 // TODO: part #1: use a for loop to replace the words in sentence 1 with
 // 'chirp' (10 points)
 
-// your code goes here
+var intVar = ' '; //variable to store and build chirps
+var words = sentence1.split(' '); //copies words from sentence into words array
+
+for (var i = 0; i < words.length; i++) { //code will run once for each word in sentence1
+  intVar += 'chirp '; //add a chirp to intVar
+}
+sentence1 = intVar.trim() + '.'; //set chirps to sentence1, remove whitespace and .
+console.log(sentence1);
 
 // TODO: part #2: use a while or do-while loop to replace the words in sentence 2
 // with 'chirp' (10 points)
 
-// your code goes here
+var count = '';
+var j = 0;
```

Comment:
this works...and...you want to be using the starting value of sentence 2 in here somewhere. Take that string, transform it, and end up with a string of chirps at the end.